cpp/ql/src/semmlecode.cpp.dbscheme

```diff
@@ -420,7 +420,7 @@ function_deleted(unique int id: @function ref);
 
 function_defaulted(unique int id: @function ref);
 
-
+member_function_this_type(int id: @function ref, int this_type: @type ref);
```

Comment:
```suggestion
member_function_this_type(unique int id: @function ref, int this_type: @type ref);
```
Some other similar relations don't have this, either (e.g., `function_return_type`), so maybe we can just defer this until we start looking at ODR-related issues in earnest. That way you don't have to re-hash the dbscheme for this change, either.

Reply:
Changing the hash isn't too hard, so I may as well do this (I'll squash, and force push the commits).